good-enough-practices-for-scientific-computing.tex

```diff
@@ -371,7 +371,7 @@ \section{Software}\label{sec:software}
   a dozen parameters.  The key motivation here is to fit the program
   into the most limited memory of all: ours. Human short-term memory
   is famously incapable of holding more than about seven (plus or
-  minus two) items at once. If we are to understand what our software
+  minus two)\fixme{LN}{add reference?} items at once. If we are to understand what our software
```

Comment:
Ref [1] there is http://psychclassics.yorku.ca/Miller/ (or http://psycnet.apa.org/journals/rev/63/2/81/), perhaps use that one instead?